ui/js/controllers/main.js

```diff
@@ -508,10 +508,10 @@ treeherderApp.controller('MainCtrl', [
         };
 
         const getNewReloadTriggerParams = function () {
-            return _.pick(
-                $location.search(),
-                ThResultSetStore.reloadOnChangeParameters,
-            );
+            const locationSearch = $location.search();
```

Comment:
When I test this locally, I'm not getting the same response from the old code as the new.  We need to handle the case where the ``prop`` does not exist in the ``locationSearch``:

old:

    Object { repo: "mozilla-inbound" }

new:

    Object { repo: "mozilla-inbound", revision: undefined, author: undefined, fromchange: undefined, tochange: undefined, startdate: undefined, enddate: undefined, nojobs: undefined }